packages/shell-api/src/database.spec.ts

```diff
@@ -84,7 +84,9 @@ describe('Database', () => {
         returnType: 'AggregationCursor',
         platforms: ALL_PLATFORMS,
         topologies: ALL_TOPOLOGIES,
-        serverVersions: ALL_SERVER_VERSIONS
+        serverVersions: ALL_SERVER_VERSIONS,
+        isDirectShellCommand: false,
```

Comment:
This is a custom "autocompleter" for arguments? And this will work with a worker thread as well cause the function will be called inside the runtime right?

Slowly getting it :).

Reply:
> This is a custom "autocompleter" for arguments?

The entry below is, yes.

> And this will work with a worker thread as well cause the function will be called inside the runtime right?

I *think* so, yes. The autocompleter in the browser also lives inside the same environment as e.g. the `ShellInternalState` object, and only the caller-facing `getCompletions()` method of the runtime is forwarded to the parent/rendering process, so this should work.